charts/argocd-image-updater/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: argocd-image-updater
 description: A Helm chart for Argo CD Image Updater, a tool to automatically update the container images of Kubernetes workloads which are managed by Argo CD
 type: application
-version: 0.2.1
+version: 0.2.2
```

As this adds a backward-compatible feature, we bump the minor version according to the SemVer spec.
```suggestion
version: 0.3.0
```